api/app/db/crud/api_access_audits.py

```diff
@@ -0,0 +1,23 @@
+""" CRUD create function for audit logs
+"""
+from app.time_utils import get_utc_now
+import logging
+from app.db.models import APIAccessAudit
+
+import app.db.database
+
+logger = logging.getLogger(__name__)
+
+
+def create_api_access_audit_log(
+        username: str,
+        success: bool,
+        path: str) -> APIAccessAudit:
```

Comment:
Method does not return anything, void equivalent should be None
```suggestion
        path: str) -> None:
```